src/ContainerCommitModal.jsx

```diff
@@ -158,7 +158,12 @@ const ContainerCommitModal = ({ container, localImages }) => {
                      </Button>
                  </>}
         >
-            {commitContent}
+            <Stack hasGutter>
```

Comment:
You can achieve the same visual result, by putting the dialogError inside the Form; I prefer it as it does not require this extra component. 

```diff
         <Form isHorizontal>
+            {dialogError && <ErrorNotification errorMessage={dialogError} errorDetail={dialogErrorDetail} onDismiss={() => setDialogError("")} />}
```

Reply:
Thanks. Moved it inside Form.